support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -168,10 +181,18 @@ function describeProduct(product: string, ratePlan: string) {
 		}
 	}
 
+	if (product === 'NationalDelivery') {
```

Comment:
Is `NationalDelivery` the same as `HomeDelivery`  - ie could they be combined?

Reply:
I am trying keep things decoupled at the moment making things easy to change going forward.